webapp/src/main/webapp/src/app/sender/sync/sender-sync-message.ts

```diff
@@ -16,4 +16,7 @@ export class SenderSyncMessage extends BaseEntity {
 
 	dateSent?: string;
 
+	eventDate?: string;
+
+
```

Comment:
Where is the other UI change to show the value?

Reply:
Did you see this comment?

Reply:
I created a Tooltip for that as we don't have space on the table